src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -189,6 +239,72 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit {
     this.inlineFormClose.emit(inlineFormCloseArgs);
   }
 
+  public moveToTop(event: Event): void {
+    event.stopPropagation();
+    this.adapterService.moveItemUp(this.elementRef, true);
+    (<HTMLElement> event.target).focus();
+  }
+
+  public handleKeyboardEvent(event: KeyboardEvent): void {
```

Comment:
What would you say about renaming these event handlers something like...
```
onReorderHandleKeyDown()
onReorderHandleBlur()
```

I had to search for these methods in the template to understand where they were coming from and what their intent was. If we rename them something more semantic, then a developer wouldn't have to go searching.

Reply:
Yup. Makes sense. Done